### PR TITLE
feat: structure storybook

### PR DESCRIPTION
**Pull request by GutierrezGo**
References 
Slack discussion: https://automotivesmg.slack.com/archives/C03GK939L3V/p1665476814092119

## Motivation and context

To have a more understandable structure and a better visualisation in our components categories. In addition to having a synchronisation between figma and storybook structures.

Please take a look between the figma files and the storybook structure and let me know if you have another suggestion/idea/disagreement.

### CODE REVIEW

File: src/components/box/index.stories.mdx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { Canvas, Meta, Story } from '@storybook/addon-docs';
 
 import Box from './index.tsx';
 
-<Meta title="Patterns/Box" component={Box} />
+<Meta title="Components/Utils/Box" component={Box} />
```
Comment (lkappeler) [inline src/components/box/index.stories.mdx:5]: Consider moving the box to the layout

**Review by lkappeler — APPROVED** [review]
Very nice thanks a lot ❤️ 
I added a list component meanwhile it goes to `Components/Data Display/List` when you merge master before merging back

File: src/components/box/index.stories.mdx (hunk 1 above)
Comment (GutierrezGo) [reply src/components/box/index.stories.mdx:5]: Yes, it makes more sense in the layout category. Thanks!

**Comment by Averethel** [comment]
Thanks for taking care of this. Keep in mind that a new storybook structure doesn't alter the functionality, so there's no need to issue a release.

**Comment by GutierrezGo** [comment]
> Thanks for taking care of this. Keep in mind that a new storybook structure doesn't alter the functionality, so there's no need to issue a release.

@Averethel yes it should be `chore` I realised that I had already pressed the button  🤦🏽‍♀️

**Pull request merged**